### PR TITLE
CamelCase > kebab-case

### PR DESCRIPTION
**Pull request by jgravois**
CamelCase project names are non-standard in the Esri GitHub organization and not allowed in [`package.json`](https://docs.npmjs.com/files/package.json#name) (on the off chance you ever wanted to publish on npm)

### CODE REVIEW

**Comment by tomwayson** [comment]
I agree, this is what it should be, but prob should wait for #36 and merge master at this point I think

**Comment by jgravois** [comment]
> should wait for #36 and merge `master`

agreed.

File: translations/en-us.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
   },
   "cache": {
     "no_data": "No data found for this address.",
-    "title": "MyStreet",
+    "title": "my-street",
     "bullet_one": "Click D.C. or L.A. navigation options above to use the application.",
-    "bullet_two": "This <a href=\"https://github.com/Esri/MyStreet\">project</a> is an <a href=\"http://emberjs.com/\">Ember App</a> that demonstrates the <a href=\"https://github.com/Esri/MyStreet/blob/master/hub-ready.md\">requirements of hub-ready</a> applications.",
+    "bullet_two": "This <a href=\"https://github.com/Esri/my-street\">project</a> is an <a href=\"http://emberjs.com/\">Ember App</a> that demonstrates the <a href=\"https://github.com/Esri/my-street/blob/master/hub-ready.md\">requirements of hub-ready</a> applications.",
```
Comment (MarkhamShofner) [inline translations/en-us.json]: we will need to change the repo name to match this URL, right?

Will we also need to change the older repo (/ArcGIS/MyStreet) that currently redirects to here.

Comment (jgravois) [reply translations/en-us.json]: > we will need to change the repo name to match this URL, right?

absolutely

> Will we also need to change the older repo (/ArcGIS/MyStreet) that currently redirects to here.

you mean (/ArcGIS/your-street)? my vote in https://github.com/ArcGIS/your-street/issues/11 is to just 🔪  that one.

Comment (MarkhamShofner) [reply translations/en-us.json]: Fine to 🔪 , double checking with Andrew on that issue you posted. Also looks like there is one more issue hanging out there that could be brought over for posterity. 

What I mean is that if you navigate to `https://github.com/ArcGIS/MyStreet` you are auto-redirected to `Esri/MyStreet`. Not a big deal, but would be nice to have all roads end in one place.

**Comment by tomwayson** [comment]
@jgravois I'll take on the merge conflicts in this since you helped out w/ #36

**Comment by jgravois** [comment]
> I'll take on the merge conflicts

works for me. but i'll be checking for stray `<<<<<<< HEAD`s after you push. 😉

**Comment by tomwayson** [comment]
NOTE: I also removed package-lock.json from .gitignore - aren't we supposed to check those in nowadays?

**Comment by jgravois** [comment]
> aren't we supposed to check those in nowadays?

i didn't wanna distract all y'all ember / lerna developers with npm@5 ✨ , but you're right. checking the package-lock.json into version control is definitely the right move.

**Comment by tomwayson** [comment]
That's very considerate. What I've been doing on my other ember projects is using yarn. We could do that here too.

**Comment by tomwayson** [comment]
Also, I guess I shoulda run tests locally before pushing, looks like there's some more clean up to do.

**Pull request closed without merging**